telemetry_components/src/main/java/com/newrelic/telemetry/SimpleSpanBatchSender.java

```diff
@@ -0,0 +1,32 @@
+package com.newrelic.telemetry;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.newrelic.telemetry.http.HttpPoster;
+import com.newrelic.telemetry.spans.SpanBatchSender;
+import com.newrelic.telemetry.spans.SpanBatchSenderBuilder;
+import java.time.Duration;
+
+public class SimpleSpanBatchSender {
+
+  public static SpanBatchSender build(String apiKey) {
+    return build(apiKey, Duration.ZERO);
+  }
+
+  public static SpanBatchSenderBuilder builder(String apiKey) {
+    return builder(apiKey);
+  }
+
+  public static SpanBatchSender build(String apiKey, Duration callTimeout) {
+    return builder(apiKey, callTimeout).build();
+  }
+
+  public static SpanBatchSenderBuilder builder(String apiKey, Duration callTimeout) {
```

Comment:
I'd like to change the metric version of this to look more like this, as well.

Reply:
I picked that up in this PR now too.